plugin/Field/FieldModuleCollectionDecoratorCustomLabelForm.php

```diff
@@ -68,27 +90,29 @@ public function getFieldByModuleAndName(string $module, string $name): Field
 	public function getAllFields(): array
 	{
 		$fields = parent::getAllFields();
-
-		foreach ($fields as $pField) {
-			$module = $pField->getModule();
-			$name = $pField->getName();
+		$cloneFields = array();
+		foreach ($fields as $key => $field) {
+			$cloneFields[$key] = clone $field;
+			$module = $cloneFields[$key]->getModule();
+			$name = $cloneFields[$key]->getName();
 			$label = $this->_fieldCustomLabels[$module][$name] ?? null;
 
 			if ($label !== null) {
-				$pField->setLabel(__($label, 'onoffice-for-wp-websites'));
+				$cloneFields[$key]->setLabel(__($label, 'onoffice-for-wp-websites'));
```

Comment:
the translation function is not necessary here, is it? _`__($x, 'onoffice-for-wp-websites')`_